### PR TITLE
Update bz2.rb

### LINKED ISSUE #2591: libbz2.so.1 is missing from /usr/local/lib64
There should be a symlink from `libbz2.so.1.0` to `libbz2.so.1`.
```
$ ls -la /usr/local/lib64/libbz2.*
lrwxrwxrwx. 1 chronos chronos    15 Jul 27  2017 /usr/local/lib64/libbz2.so.1.0 -> libbz2.so.1.0.6
-rwxr-xr-x. 1 chronos chronos 69520 Aug 22 13:52 /usr/local/lib64/libbz2.so.1.0.6
```
This causes an error when trying to build Weston.
```
/usr/local/bin/ld: /lib64/libbz2.so.1: unknown type [0x13] section `.relr.dyn'
/usr/local/bin/ld: /lib64/libbz2.so.1: unknown type [0x13] section `.relr.dyn'
/usr/local/bin/ld: warning: libbz2.so.1, needed by /usr/local/lib64/libfontconfig.so.1, not found (try using -rpath or -rpath-link)
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompress'
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompressEnd'
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompressInit'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:4686: weston-simple-im] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/4.0.0.tar.gz.dir/weston-4.0.0'
make: *** [Makefile:3570: all] Error 2
weston failed to install: `make V=0 -j4` exited with 2
```

### PR DESCRIPTION
**Pull request by superloach**
Fixes #2591

### CODE REVIEW

**Comment by uberhacker** [comment]
Thank you for the update @superloach.  Can you please bump the version since we will need new binaries?
```
- version '1.0.6'
+  version '1.0.6-1'
```

**Comment by superloach** [comment]
There we go 👍

**Comment by uberhacker** [comment]
Thanks.  Is bzip.org dead?  Can you download the source?  Maybe we can use this as an alternative?  https://fossies.org/linux/misc/bzip2-1.0.6.tar.xz

File: packages/bz2.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@ def self.install
     system "mkdir", "-p", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "cp", "-p", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "ln", "-s", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1.0"
+    system "ln", "-s", "libbz2.so.1.0", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1"
```
Comment (uberhacker) [inline packages/bz2.rb]: Actually, now that I saw it a second time, can you please change `#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}` to `#{CREW_DEST_LIB_PREFIX}`?

**Comment by JL2210** [comment]
Finally, a mirror for bzip2! Yes, bzip.org is dead, after long last.

File: packages/bz2.rb (hunk 1 above)
Comment (uberhacker) [reply packages/bz2.rb]: Also, I noticed you could change `#{CREW_DEST_DIR}/usr/local` to `#{CREW_DEST_PREFIX}`.

**Comment by JL2210** [comment]
I'll use a mirror on `kernel.org`.

**Comment by JL2210** [comment]
Well, the `fossies.org` alternative is probably better. Thanks @uberhacker!

**Comment by JL2210** [comment]
I could create a simple github page for it.

**Comment by JL2210** [comment]
Actually, you are probably better off doing it than I am, @superloach. Give it a try.

**Comment by superloach** [comment]
Alright, I can do that now. Thanks for the help!

**Comment by superloach** [comment]
How's that @JL2210?

**Comment by JL2210** [comment]
Good!

**Comment by JL2210** [comment]
XZ is always better than GZ. Good job!

**Pull request merged**